cmd/demo/local-testnet.sh

```diff
@@ -0,0 +1,113 @@
+CURRENT_DIR=$(pwd)
+WORKING_DIR="$CURRENT_DIR"
+TESTNET_DIR=$WORKING_DIR/testnet
+TESTNET_OUTPUT_DIR=$TESTNET_DIR/testnet-local
+SCRIPTS_DIR=mx-chain-go/scripts/testnet
+VARIABLES_PATH=$SCRIPTS_DIR/variables.sh
+OBSERVERS_PATH=$SCRIPTS_DIR/include/observers.sh
+VALIDATORS_PATH=$SCRIPTS_DIR/include/validators.sh
+ENABLE_EPOCH_DIR=$TESTNET_DIR/mx-chain-go/cmd/node/config/enableEpochs.toml
+SANDBOX_NAME=sandbox
+
+cloneDependencies(){
+  if [ -d "$TESTNET_DIR" ]; then
+    rm -rf $TESTNET_DIR
+  fi
+
+  mkdir "$TESTNET_DIR"
+
+  git clone https://github.com/multiversx/mx-chain-go "$TESTNET_DIR/mx-chain-go"
+  cd $TESTNET_DIR/mx-chain-go
+  git checkout rc/v1.6.0
```

Comment:
better use commit hashes? for all repos (mx-chain-go, mx-chain-deploy-go, mx-chain-proxy-go), since we will be sure that this demo will always work

Reply:
or better, tags, we can now create the v1.6.0 tag. We never remove tags, only releases